src/Sylius/Component/Order/Model/Adjustment.php

```diff
@@ -104,8 +89,6 @@ public function getAdjustable()
         if (null !== $this->orderItem) {
             return $this->orderItem;
         }
-
-        return null;
```

Comment:
This should not be removed. I know it's default behavior, but let's be explicit here.